lib/graphql/schema/argument.rb

```diff
@@ -250,7 +258,7 @@ def coerce_into_values(parent_object, values, context, argument_values)
           value = values[arg_key]
         elsif default_value?
           has_value = true
-          value = default_value
+          value = default_value(context)
```

Comment:
@rmosolgo I'm looking at this a little more and I realize I'm confused. We call `coerce_input` on this value a few lines down, which at best is a no-op but (since this value should already be specified as ruby-land-objects and not graphql-syntax-input) would most likely fail entirely? What am I missing?

Reply:
I'm admittedly fuzzy on the details but the _intent_ here is to prepare some GraphQL input for handling by the application. (Eventually, `argument_values` here will populate a `GraphQL::Execution::Interpreter::Arguments` instance.) So, unless I've misunderstood, I'd expect `type.coerce_input(...)` below to prepare a GraphQL value for handling by Ruby. (Eg, turning enum strings into their `value: ...`s.) 

But maybe that's the wrong approach for `default_value`s, if they're already Ruby-style!